src/patternfly/components/Select/select.scss

```
@@ -234,6 +234,12 @@
   .pf-c-select__toggle-clear + .pf-c-select__toggle-button {
     padding-left: var(--pf-c-select__toggle-button--PaddingLeft);
   }
+
+  .pf-c-select__toggle-arrow {
+    .pf-m-top.pf-m-expanded > & {
```

Can you scope the `.pf-m-top.pf-m-expanded` selector to `.pf-c-select` so that, in the case that the select is nested inside of something else with `.pf-m-top.pf-m-expanded`, this CSS doesn't accidentally apply? See the dropdown for a similar example

https://github.com/patternfly/patternfly-next/blob/master/src/patternfly/components/Dropdown/dropdown.scss#L248-L257

```suggestion
    .pf-c-select.pf-m-top.pf-m-expanded > & {
```